.github/workflows/lint-python.yml

```diff
@@ -1,10 +1,22 @@
 name: lint_python
-on: [pull_request, push]
+on:
+  push:
+    branches: [master]
+  pull_request:
 jobs:
   lint_python:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-      - run: pip install tox
-      - run: tox -e lint
```

Comment:
Kept the run command as install tox & tox -e lint.

This allow to run the same linter locally